variables.tf

```diff
@@ -8,7 +8,7 @@ variable "centos_image" {
     default = "CentOS 7.7"
 }
 variable "ssh_key_name" {
-    default = "bootstrap"
+    default = "kennedy"
```

Comment:
Revert this please.

Reply:
Is there a bootstrap key I should be using for this? I have just been changing it to my key to use, but I often forget to exclude that change as above.

Reply:
Yes, it's on fir and fair to copy it and use it.